CHANGELOG.md

```diff
@@ -3,6 +3,21 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased][unreleased]
+### Added
+- Support for [Lists API](https://docs.courier.com/reference/lists-api) by @tk26
+  - `GET /lists` with `courier.GetLists` method
+  - `GET /list/{list_id}` with `courier.GetList` method
+  - `PUT /list/{list_id}` with `courier.PutList` method
+  - `DELETE /list/{list_id}` with `courier.DeleteList` method
+  - `PUT /list/{list_id}/restore` with `courier.RestoreList` method
+  - `GET /list/{list_id}/subscriptions` with `courier.GetListSubscriptions` method
+  - `PUT /list/{list_id}/subscriptions` with `courier.PutListSubscriptions` method
+  - `POST /list/{list_id}/subscriptions` with `courier.PostListSubscriptions` method
+  - `PUT /lists/{list_id}/subscriptions/{recipient_id}` with `courier.PutListSubscriptionsRecipient` method
+  - `DELETE /lists/{list_id}/subscriptions/{recipient_id}` with `courier.DeleteListSubscriptionsRecipient` method
```

Comment:
For these last 2 we have been using `subscribe` and `unsubscribe` language. So maybe these should be `courier.SubscribeList` and `courier.UnsubscribeList`? Or `courier.ListSubscribe` and `courier.ListUnsubscribe`.

Reply:
I like that

Reply:
Going with `ListSubscribe` and `ListSubscribe`